lib/host_patterns.rb

```diff
@@ -0,0 +1,13 @@
+module HostPatterns
+  DEFAULT_HOST_PATTERNS = [
+    /forms\.service\.gov\.uk/,
```

Comment:
I think `forms.service.gov.uk` always redirects to `www.forms.service.gov.uk`? I'm not sure how that happens thought...

Reply:
Ok, I've amended the list based on this conversation -- hopefully that's the final touch!